Header header
uint32 number_of_objects
Object[] obj